### DISCUSSION TITLE
Jellyfin 支持中文字体

### DISCUSSION THREAD

**Discussion by jannson** [istoreos/istoreos#644] [General]
引用来自微信群的反馈：

字体下载路径：
https://raw.githubusercontent.com/CodePlayer/webfont-noto/master/release/NotoSansCJKsc-hinted-standard.zip

文件路径：
/mnt/nvme0n1-5/docker/volumes/aaa8e79a8db0ecb1ebf77909e10900d3101bf407e131d5d722d688c99f6c074b/_data/fonts

帖子参考：
https://blog.lishun.me/docker-jellyfin-chinese-fonts

**Comment by jjm2473** [comment]
RK3568系列的jellyfin镜像和ARS2镜像都是自带中文字体的，不需要这些操作。x86的可能需要

**Reply by nwdytr** [reply]
x86
有没有……小白能看懂的操作办法……
